src/App.jsx

```diff
@@ -1,107 +1,18 @@
 import './App.scss';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+const postsWithComments = postsFromServer.map(post => ({
+  ...post,
+
+  comments: commentsFromServer.filter(comment => comment.postId === post.id),
```

Comment:
Consider finding a user and rendering user information
<img width="959" alt="Screenshot 2023-10-30 at 16 39 18" src="https://github.com/mate-academy/react_static-list-of-posts-js/assets/135232001/71d44b19-7172-41cb-b74c-99863b8652f9">